benchmarking/benchmark.py

```diff
@@ -30,7 +30,7 @@
 N_SPLITS = 5
 # N_REPEATS - number of test repetitions - for getting average time score
 N_REPEATS = 5
-N_CORES = [i for i in range(1, cpu_count() + 1)]
+N_CORES = [i for i in range(1, cpu_count() + 1)][:8]
```

Comment:
Why? If we have more cores, then good

Reply:
Oh right. It was for the purpose of creating a presentation. I'll fix it